api/src/test/java/org/pmiops/workbench/billing/FreeTierBillingServiceTest.java

```diff
@@ -229,8 +231,8 @@ public void checkFreeTierBillingUsage_combinedProjectsExceedsLimit() {
       DbWorkspaceFreeTierUsage usage = workspaceFreeTierUsageDao.findOneByWorkspace(ws);
       assertThat(usage.getUser()).isEqualTo(user);
       assertThat(usage.getWorkspace()).isEqualTo(ws);
-      assertThat(usage.getCost())
-          .isCloseTo(expectedCosts.get(ws.getWorkspaceId()), withinPercentage(0.001));
+
+      assertCloseEnough(usage.getCost(), expectedCosts.get(ws.getWorkspaceId()));
```

Comment:
Does this automatically pickup the percentage value? Otherwise this is a change of behavior.

Reply:
Edit: I learned later in the PR that this is not a provided method, but a helper we have written. We should make that explicit somehow, or just use the `assertWithinPercentage` method. (I don't know if the abstraction gives us much, cause we've already abstracted the variable so changes only need to happen in one place regardless)

Reply:
The default value is a class constant, so we should be free to raise the abstraction level from "within CLASS_CONSTANT" to "close enough for the purposes of this class". It's very cheap to make private methods, and pretty much any time you can shrink an argument list and maintain an appropriate abstraction, that's always a win in my book.

There's also still the possibility of someone passing in the wrong value in the 3-arg form.